rust/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "xicore"
+name = "xi-core"
```

Comment:
I'm fine with this, but AppDelegate.swift will need to have its reference to this binary changed as well, and, I'm guessing, the build script that copies the binary into the app.